nss-day-cohort-48/fun-website-1-lemonade-stand
# Added my html-info page

I made a rough draft for the html-info page as well as removed the template.html file. 

STEPS TO TEST:

1. Verify html-info.html includes actual code.
2. Verify template.html is no longer a file in the repo
